docs/_embed/partner_api.md

```diff
@@ -669,6 +861,19 @@ All errors will be JSON object with a String array of errors:
 ```
 
 ## Changelog
+- 2023-05-25
+
+  - Added query parameters to `v1/apps`
+
+    - The endpoint `/v1/apps` supports query parameters `title_search`, `title_starts_with`, and `category` for more refined app search results
+
+  - Added query parameter to `v1/zaps`
+
+    - The endpoint `/v1/zaps` supports the query parameters `is_in_zap_template_with`.
```

Comment:
This will be under `/v1/apps`, not `/v1/zaps`. It's not yet finished either, so feel free to either leave it out of the MR for now, or make a note that this MR cannot be merged until https://zapierorg.atlassian.net/browse/TPS-2402 and https://zapierorg.atlassian.net/browse/TPS-2403 are Done.

Reply:
Finished! Still should be under `/v1/apps` with the other new query params.

You can add `get_params` to `/v1/zaps` though!